garbage-collection-configuration.md

```diff
@@ -36,6 +37,12 @@ The `CENTRAL` garbage collection mode is no longer supported. The `DISTRIBUTED`
 
 For information on changes in previous releases, refer to earlier versions of this document using the _TIDB version selector_ in the left hand menu.
 
+## Changes in TiDB 6.1.0
+
+In versions before TiDB 6.1.0, the internal transaction does not affect the GC safe point. Since TiDB 6.1.0, the internal transaction's startTS is considered when calculating the GC safe point, to resolve the access problem of cleaned data. When the internal transaction is too long, the safe point will be blocked for a long time, which affect the performance of application.
```

Comment:
```suggestion
Before TiDB 6.1.0, the transaction in TiDB does not affect the GC safe point. Since v6.1.0,  TiDB considers the startTS of the transaction when calculating the GC safe point, to resolve the problem that the data to be accessed has been cleared. If the transaction is too long, the safe point will be blocked for a long time, which affects the application peformance.
```